DataFormats/HcalRecHit/test/HcalRecHitDump.cc

```diff
@@ -55,13 +55,26 @@ namespace {
     }
 
     void printRecHitAuxInfo(std::ostream& s, const HFPreRecHit& i,
-                            const std::vector<int>& bits)
+                            const std::vector<int>& bits, bool)
     {
     }
 
     void printRecHitAuxInfo(std::ostream& s, const HBHERecHit& i,
-                            const std::vector<int>& bits)
+                            const std::vector<int>& bits, const bool plan1)
     {
+        if (plan1 && i.isMerged())
+        {
+            // This is a "Plan 1" combined rechit
+            std::vector<HcalDetId> ids;
+            i.getMergedIds(&ids);
+            const unsigned n = ids.size();
+            s << "; merged " << n << ": ";
+            for (unsigned j=0; j<n; ++j)
+            {
+                if (j) s << ", ";
```

Comment:
implicit conversions should be better avoided 
if (j!=0) ...
But ok, this is test code, I won't be strict on it...